test/integration/quic_http_integration_test.cc

```diff
@@ -1390,6 +1390,28 @@ TEST_P(QuicHttpIntegrationTest, DeferredLoggingWithInternalRedirect) {
   EXPECT_EQ(/* RESP(test-header) */ metrics.at(21), "-");
 }
 
+TEST_P(QuicHttpIntegrationTest, InvalidTrailer) {
+  initialize();
+  // Empty string in trailer key is invalid.
+  Http::TestRequestTrailerMapImpl request_trailers{{"", "foo"}};
+  Http::TestResponseTrailerMapImpl response_trailers{{"a", "b"}, {"c", "d"}};
```

Comment:
Awesome, thanks for tracking down a testable case!
Do we need response trailers or can we nix them?

Reply:
Meant to remove those, thanks for the catch!